.github/workflows/ci.yml

```diff
@@ -36,9 +36,19 @@ jobs:
       report-jobs: ${{ steps.project-jobs.outputs.report-jobs }}
     steps:
       - uses: 'actions/checkout@v4'
-        name: 'Checkout'
+        name: "Checkout (${{ github.event.pull_request.base.ref || github.base_ref }})"
+        with:
+          ref: ${{ github.event.pull_request.base.ref || github.base_ref }}
+          fetch-depth: 500
```

Comment:
~3.5 weeks of commits history.

Reply:
Is this a final depth? Why did you choose 500?

Reply:
Nope, not final one. Picked by combining:
- recommendation for the depth in CI (found GitLab recommendation to be max 1000)
- checking repo stats (~150 merged PRs withing one week) and ensuring the number covers 2-3 weeks of merges